vampire_app/app.js

```diff
@@ -15,37 +29,339 @@
 // INSERT USING MONGOOSE
 // ### Add the vampire data that we gave you
 
+const vampireData = require('./populateVampires')
+
+const Vampire = require('./models/vampire')
+
+// Vampire.collection.insertMany(vampireData, (err, data) => {
+// 	if(err) { 
+// 		console.log(err);
+// 	} else {
+// 		console.log('added provided vampire data');
+// 	}
+// })
+
 // ### Add some new vampire data
 
+// Vampire.create(
+// {name: 'Ambroze Gaug',
+//  hair_color: 'white',
+//  eye_color: 'gray',
```

Comment:
`eye_color` ignored/not added, as it is missing from your model.  (this applies to anywhere you try to use/add `eye_color`)